CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ option(ENABLE_PACKAGE_TAR "Enable package artifacts to tar." OFF)
 option(ENABLE_CREATE_GIT_HOOKS "Enable create git hooks." ON)
 option(ENABLE_INCLUDE_WHAT_YOU_USE "Enable include-what-you-use find  nouse include files" OFF)
 
+SET(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
```

Comment:
This `CMAKE_EXPORT_COMPILE_COMMANDS` option could be specified in cmake configuration step and need not to always turn it on. please cleanup it.

Reply:
please let the CI run.

Reply:
@fansehep hi, according to yixinglu's response, you can simply remove the added `CMAKE_EXPORT_COMPILE_COMMANDS` configuration statement. Currently, it is set to true, but as per yixinglu's suggestion, it is already configurable elsewhere, so there is no need to hardcode it here.

Reply:
thanks for your reminders, but from the CI compile output, the mold linker link error :(

Reply:
Nebula CI sometimes needs to run a few more times to pass. I will rerun it a few more times. Wait for my good news 🥺.

Reply:
> thanks for your reminders, but from the CI compile output, the mold linker link error :(

Perhaps you should add some link dependencies, as commented in this issue. #5840

Reply:
@fansehep To fix the failure of CI, u'd better to check where the target `$<TARGET_OBJECTS:function_manager_obj>` is used in the whole code base, and add the the `${PROXYGEN_LIBRARIES}` dependency for there.

Reply:
@fansehep CI has been fixed, shall we go on?